cursor_pagination.py

```diff
@@ -39,28 +39,55 @@ def __repr__(self):
 
 class CursorPaginator(object):
     delimiter = '|'
+    none_string = '::None'
     invalid_cursor_message = _('Invalid cursor')
 
     def __init__(self, queryset, ordering):
-        self.queryset = queryset.order_by(*ordering)
+        self.queryset = queryset.order_by(*self._nulls_ordering(ordering))
         self.ordering = ordering
 
+    def _nulls_ordering(self, ordering, from_last=False):
+        """
+        This clarifies that NULL value comes at the end in the sort.
+        When "from_last" is specified, NULL value comes first since we return the results in reversed order.
+        """
+        nulls_ordering = []
+        for o in ordering:
```

Comment:
Really picky I know, but could we name this something other than `o` please.

Reply:
Renamed!